psbt/src/roles/extractor.rs

```diff
@@ -36,6 +36,6 @@ where
             }
         }
 
-        Ok(builder.build())
+        Ok(builder.build()?)
```

Comment:
There has to be a better way to do this

Reply:
I think putting the `?` inside of the `Ok` feels kind of ugly but I found other code doing the same thing

Reply:
It's a pretty common pattern for converting error types